.github/ISSUE_TEMPLATE/testing.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Testing
+about: Suggest covering some code with tests
+title: ''
+labels: 'type: testing'
+assignees: ''
+
+---
+
+### Involved Module
+- [ ] UI
+- [ ] EvoSuite
+- [ ] LLM
+- [ ] Kex
+- [ ] Other (please explain)
+
+### What part of the code needs testing?
+Please point to a specific component that needs testing.
+
+### How error-prone is this code?
+Does it contain any complex and error-prone logic?
+
+### How critical is this code?
+Would bugs here strongly affect user experience?
+
+### How much time and effort will the testing take?
+Is it a 30-minute task or does it require hours of testing? Do we need to write any fakes, mocks, or stubs?
```

Comment:
please, separate time spending question; and testing stategy + frameworks question